### PR TITLE
qa/suites: escape the parenthesis of the whitelist text

### PR DESCRIPTION
**Pull request by tchaikov**
the ceph-disk change addresses the failures of http://pulpito.ceph.com/kchai-2017-07-30_05:19:08-ceph-disk-wip-kefu-testing-distro-basic-vps/

### CODE REVIEW

File: qa/workunits/ceph-disk/ceph-disk.sh [hunk 1]
```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
-if [ -f $(dirname $0)/../../standalone/ceph-helpers-root.sh ]; then
-    source $(dirname $0)/../../standalone/ceph-helpers-root.sh
+if [ -f $(dirname $0)/../../standalone/ceph-helpers.sh ]; then
+    source $(dirname $0)/../../standalone/ceph-helpers.sh
```
Comment (liewegas) [inline qa/workunits/ceph-disk/ceph-disk.sh]: shouldn't this switch back to ../../workunits/ceph-helpers-root.sh?  (I'm guessing the -root version is different?)

**Comment by tchaikov** [comment]
@liewegas fixed and repushed.

File: qa/workunits/ceph-disk/ceph-disk.sh [hunk 2]
```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
-if [ -f $(dirname $0)/../../standalone/ceph-helpers-root.sh ]; then
-    source $(dirname $0)/../../standalone/ceph-helpers-root.sh
+if [ -f $(dirname $0)/../ceph-helpers.sh ]; then
+    source $(dirname $0)/../ceph-helpers.sh
```
Comment (liewegas) [inline qa/workunits/ceph-disk/ceph-disk.sh]: with -root too right?

Comment (tchaikov) [inline qa/workunits/ceph-disk/ceph-disk.sh]: ahh, right! forgot to update the filename!

fixed and pushed again.

**Comment by tchaikov** [comment]
- http://pulpito.ceph.com/kchai-2017-08-02_02:03:08-ceph-disk-wip-kefu-testing2-distro-basic-vps/
- http://pulpito.ceph.com/kchai-2017-08-01_17:41:46-rados-wip-kefu-testing-distro-basic-mira/

**Pull request merged**